utils/styleguide/styleguide.base.config.js

```diff
@@ -13,7 +13,7 @@ const basePathName = path.basename(path.resolve('./'));
 const googleTrackingId = 'UA-970836-25';
 
 const defaultStyleguideConfig = {
-  title: 'Zendesk Garden',
+  title: `${packageManifest.name} - Zendesk Garden`,
```

Comment:
this isn't going to result in a great title: "@zendeskgarden/[name] - Zendesk Garden"

Prefer following the format seen under https://garden.zendesk.com/css-components/avatars/: "[name] / React Components / Zendesk Garden"

Reply:
I updated this to use a capitalized version of the baseName (`/packages/[baseName]`). This looks a lot better.